src/web/mjs/connectors/anikiga.mjs

```diff
@@ -1,12 +0,0 @@
-import WordPressMadara from './templates/WordPressMadara.mjs';
-
-export default class Anikiga extends WordPressMadara {
```

Comment:
Website is still working for me, don't remove

Reply:
Oh I thought these didn't stick when I squashed. I'll remove this. 

That being said, I think these two files are a mistake? They're just dupes of two other files with slightly different casing. On a case insensitive file system like Windows you can't have both and so git obviously sees one or the other deleted.

Reply:
Renaming them on your end to fit case on `master` branch may solve the conflict

Reply:
I can't rename to match because both files exist in 'master'. So I'm always either removing one or the other. Like I said I can get this uncommitted. But I don't think it's intentional that these dupe files are here.